query-node/mappings/content-directory/entity/create.ts

```diff
@@ -216,8 +216,7 @@ async function createVideo(
   video.description = p.description
   video.duration = p.duration
   video.hasMarketing = p.hasMarketing
-  // TODO: needs to be handled correctly, from runtime CurationStatus is coming
-  video.isCurated = p.isCurated || true
+  video.isCurated = p.isCurated || false
```

Comment:
To convert truthy/falsy to boolean I find this syntax more comprehensible, and should convert falsy -> false and truthy -> true which is what is intended here?

```suggestion
  video.isCurated = !!p.isCurated
```

Reply:
And if p.isCurated is boolean, why not just `video.isCurated = p.isCurated`

Reply:
Mokhtar when I start to implement the curation status it wasn't clear that if we would have a simple boolean field or some struct which would have more information. That's why I setting to a default value in case if there is no such field on the runtime. But later I have missed this code :(